src/TestSuite/Constraint/Response/HeaderNotContains.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @since         3.7.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite\Constraint\Response;
+
+/**
+ * Constraint for ensuring a header does not contain a value.
+ *
+ * @internal
+ */
+class HeaderNotContains extends HeaderEquals
```

Comment:
Should this extend HeaderContains instead? HeaderEquals is a full comparison.

Reply:
Yes it should. Thank you.